clash-ghc/src-ghc/Clash/GHC/LoadModules.hs

```diff
@@ -159,6 +159,7 @@ import           Clash.Annotations.BitRepresentation.Internal
   (DataRepr', dataReprAnnToDataRepr')
 
 import           Clash.Signal.Internal
+import Data.Foldable (toList)
```

Comment:
This should be grouped with the other imports from `base`

Reply:
That's what I get for using an IDE ;)

Reply:
:exploding_head: